core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -34,17 +34,20 @@ oppia.directive('collectionSummaryTile', [
         getThumbnailIconUrl: '&thumbnailIconUrl',
         getThumbnailBgColor: '&thumbnailBgColor',
         isLinkedToEditorPage: '=?isLinkedToEditorPage',
-        getCategory: '&category'
+        getCategory: '&category',
+        isPlaylistMode: '&playlistMode',
+        showLearnerDashboardIcons: '&showLearnerDashboardIcons'
```

Comment:
I feel like my comments are getting lost somewhere. I thought we were going to do showLearnerDashboardIconsIfPossible (or perhaps showLearnerDashboardIconsIfLoggedIn)?

Reply:
I have changed this. Maybe it some bug with GitHub. Now that I'm seeing this it is showing this as outdated.